app/controllers/notices-setup.controller.js

```diff
@@ -205,9 +205,9 @@ async function viewNoticeType(request, h) {
 }
 
 async function viewPreviewReturnForms(request, h) {
-  const { sessionId } = request.params
+  const { sessionId, returnId } = request.params
```

Comment:
```suggestion
  const { sessionId, returnLogId } = request.params
```

Reply:
I am an idiot. We do have a UUID in the table now, and it is called `return_id`. I forgot we made this change 🤦